opencl-aot/CMakeLists.txt

```diff
@@ -73,13 +73,37 @@ link_directories(${CMAKE_CURRENT_BINARY_DIR}/lib
         )
 file(GLOB TARGET_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/source/*.cpp)
 
-add_llvm_tool(opencl-aot ${TARGET_SOURCES})
+set(OPENCL_AOT_PROJECT_NAME opencl-aot)
```

Comment:
please insert the following code before line 8 of the file:
if( (OpenCL_INCLUDE_DIR AND OpenCL_LIBRARY) )
  find_package(OpenCL)
endif()

It will make cmake actually accept OpenCL_INCLUDE_DIR AND OpenCL_LIBRARY variables passes in cmake command line. Finally you will avoid downlaoding and rebuild OpenCL ICD loader

Reply:
Thanks, fixed